go/billing/tests/test_admin.py

```diff
@@ -72,3 +75,20 @@ def test_account_admin_view(self):
         self.assertContains(response, 'Description')
         self.assertContains(response, 'Credit balance')
         self.assertContains(response, 'Last topup balance')
+        self.assertContains(response, 'Is developer')
+
```

Comment:
Okay so I think I've figured out a good way of testing (writing this down for future reference). Call the admin function directly, passing a `queryset` that you get in the usual manner, and an instance of an empty mock object as `request`.

Example of the test here: https://github.com/praekelt/vumi-go/pull/1195/files#diff-c2e7d22d458eb41085f772dc23e9f732R104